File: src/main/java/sharev/account/repository/OauthAccountRepository.java
package sharev.account.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import sharev.account.entity.OauthAccount;
import sharev.account.entity.OauthAccount.OauthAccountId;

public interface OauthAccountRepository extends JpaRepository<OauthAccount, OauthAccountId> {
}

File: src/main/kotlin/sharev/account/adapter/outbound/jpa/repository/OAuthAccountRepository.kt
package sharev.account.adapter.outbound.jpa.repository

import org.springframework.data.jpa.repository.JpaRepository
import sharev.account.adapter.outbound.jpa.entity.OAuthAccountJpaEntity
import sharev.account.adapter.outbound.jpa.entity.OAuthAccountJpaEntityId

interface OAuthAccountRepository : JpaRepository<OAuthAccountJpaEntity, OAuthAccountJpaEntityId> {
}
